Decsys/Data/Entities/Component.cs

```diff
@@ -3,29 +3,23 @@
 
 namespace Decsys.Data.Entities
 {
-    public class Component
+    //[Obsolete]
+    public class Component : BaseComponent
     {
+
         /// <summary>
         /// DO NOT USE. Only provided for ORM use.
         /// </summary>
         [Obsolete]
         public Component() { }
-
-        /// <summary>
-        /// Create a Component of the specified type.
-        /// </summary>
-        /// <param name="type">The component type.</param>
-        public Component(string type)
+        //Call base constructor for service layer
```

Comment:
yes please. not sure the comment is needed, but the ctor below should call `base(type)` and then have an empty body.